apps/mobile/src/styles/global-styles/spacings.ts

```diff
@@ -0,0 +1,10 @@
+const SPACINGS = {
+	xs: 8,
+	sm: 10,
+	md: 12,
+	lg: 14,
+	xl: 25,
+	xxl: 44,
+};
+
+export { SPACINGS };
```

Comment:
It's not styles, let's move it to constants. And just "SPACING"

Reply:
This should be in a shared package I guess because on the front-end these values can be used as well.
Also, lets add some extra ones and make them more consistent
```
const SPACING = {
  xxxs: 2,
  xxs: 4,
  xs: 8,
  sm: 12,
  md: 16,
  lg: 24,
  xl: 32,
  xxl: 48,
} as const;
```

Reply:
I don't think we will use this approach for front-end. In front-end we will use module css for each screen, so let's keep it on mobile side only